lib/matplotlib/dates.py

```diff
@@ -1482,52 +1510,9 @@ def __init__(self, base=1, month=1, day=1, tz=None):
         Mark years that are multiple of base on a given month and day
         (default jan 1).
         """
-        super().__init__(tz)
-        self.base = ticker._Edge_integer(base, 0)
-        self.replaced = {'month':  month,
```

Comment:
overall this seems fine.  I suspect you need a minor API note that says `self.replaced` has been dropped.  This should probably never have been public in the first place, as I can't see why a user would do anything with it, but...

Reply:
Is it worth putting in the a property what warns on get/set?  On one hand, most likely no one is using this, on the other hand it is not much work to make sure we do not needlessly break anyone who is trying to access this.